packages/wrangler/src/api/dev.ts

```diff
@@ -32,18 +32,32 @@ export async function unstable_dev(script: string, options: DevOptions) {
 		fetch: (init?: RequestInit) => Promise<Response | undefined>;
 	}>((resolve) => {
 		//lmao
+		let pidToKill: number;
 		return new Promise<Awaited<ReturnType<typeof startDev>>>((ready) => {
 			const devServer = startDev({
 				script: script,
 				...options,
 				local: true,
-				onReady: () => ready(devServer),
+				onReady: (pid) => {
+					if (pid) {
+						pidToKill = pid;
+					}
```

Comment:
Out of curiosity...
What is the scenario where `pid` is falsy but `pidToKill` has something non-falsy in it that should not be overridden?

Reply:
this is just me being overly defensive with my programming 😅 
I guess I could just blindly do `pidToKill = pid;` even if `pid` doesn't exist

Reply:
Defensive on one hand but potentially hiding a bug on the other? How about asserting that these are defined?